### PR TITLE
Revamp Kubernetes docs

### PR DESCRIPTION
**Pull request by hausdorff**

### CODE REVIEW

File: quickstart/kubernetes/faq.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+their application.
+
+* Helm 2 Charts are managed by an in-cluster API server, called Tiller, rather than the official
+  Kubernetes API server. This makes many things (_e.g._, RBAC) harder, because the API server is run
+  with a single `ServiceAccount`, which typically has global read/write access to the cluster.
+
+  Pulumi requires no server-side component. Just like `kubectl`, it uses the official [Kubernetes go
+  client library][client-go] to talk directly to the API server. It's appropriate to drop Pulumi in
+  anywhere you already use `kubectl`.
+* The Pulumi CLI presents information-rich status updates, giving a notion of progress as resources
```
Comment (lukehoban) [inline quickstart/kubernetes/faq.md]: Many of these points seem to be about `kubectl` but this section is nominally about `helm`.

File: quickstart/kubernetes/faq.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+**Yes!** Pulumi uses the official [Kubernetes Go client library][client-go]. It
+will work pretty much anywhere `kubectl` works.
+
+## Does Pulumi support the entire Kubernetes API?
+
+**Yes!** It's generated from the Kubernetes OpenAPI spec, so anything that exists in the Kubernetes
+API can be used from Pulumi. Notably, we support even alpha and beta APIs.
+
+## Can I use Pulumi without the Pulumi SaaS?
```
Comment (lukehoban) [inline quickstart/kubernetes/faq.md]: I believe we refer to this as the Pulumi Console or Pulumi Service in our docs.

File: quickstart/kubernetes/faq.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+**Yes!** The SaaS performs several functions that are useful, such as coordinating so that multiple
+users don't simultaneously update the same resources.
+
+If you don't want or need these things, then you can ask Pulumi to store the state file somewhere
+else (_e.g._, in a local file). See documentation for [`pulumi login`][login].
+
+## Does the Pulumi SaaS use credentials in my [kubeconfig file][kubeconfig] to deploy my app?
```
Comment (lukehoban) [inline quickstart/kubernetes/faq.md]: We have similar FAQ entries for Pulumi generally on the main FAQ. Suggest redirecting there for detail, and leaving just the kubernetes specific parts of the answer here.

File: quickstart/kubernetes/faq.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+---
+
+## Does Pulumi work with my [kubeconfig file][kubeconfig]?
+
+**Yes!** Pulumi uses the official [Kubernetes Go client library][client-go]. It
+will work pretty much anywhere `kubectl` works.
+
+## Does Pulumi support the entire Kubernetes API?
+
+**Yes!** It's generated from the Kubernetes OpenAPI spec, so anything that exists in the Kubernetes
```
Comment (lukehoban) [inline quickstart/kubernetes/faq.md]: It => the Pulumi Kubernetes package or provider?

File: quickstart/kubernetes/faq.md [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+API can be used from Pulumi. Notably, we support even alpha and beta APIs.
```
Comment (lukehoban) [inline quickstart/kubernetes/faq.md]: Remove “we”.

File: quickstart/kubernetes/faq.md [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+
+## Can I use Pulumi without the Pulumi SaaS?
+
+**Yes!** The SaaS performs several functions that are useful, such as coordinating so that multiple
+users don't simultaneously update the same resources.
+
+If you don't want or need these things, then you can ask Pulumi to store the state file somewhere
+else (_e.g._, in a local file). See documentation for [`pulumi login`][login].
```
Comment (lukehoban) [inline quickstart/kubernetes/faq.md]: Cross reference other docs here

**Comment by joeduffy** [comment]
Love it 💯

**Pull request merged**